Sources/xcproj/PBXBuildFile.swift

```diff
@@ -9,6 +9,7 @@ final public class PBXBuildFile: PBXObject {
     public var fileRef: String?
 
     /// Element settings
+    // sourcery: notEquatableDictionary
```

Comment:
Is this type not supported by Sourcery?

Reply:
Sourcery doesn't know anything about how to generate the models. It uses the templates that you specify. I could have mad the template more complex by traversing this type and checking if it's a non-equatable dictionary, but it was simpler using annotations.

Reply:
I meant don't we want to check for this value as well? Otherwise we're losing functionality

Reply:
Oh, I see. You are checking for it, but using this tag to differentiate it in the template. Got it